### DISCUSSION TITLE
广东第二师范学院

### DISCUSSION THREAD

**Discussion by Gdei-hero** [CollegesChat/university-information#47] [University]
1、宿舍是上床下桌吗
花都校区学生宿舍是四人间上床下桌，教师公寓则是上下床，但是二房甚至三房一厅，条件较好。海珠校区5栋新宿舍四人间上床下桌，1栋翻新宿舍六人间上床下桌，2、3、4则是上下铺，旧宿舍，条件相对较差。
2、教室和宿舍有没有空调
去年海珠校区宿舍区完成了空调安装工程，自此海珠与花都校区的教室与宿舍均有空调，且海珠校区的空调还可以制热（
3、有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远
独立卫浴。但是热水一个月能坏一两次，热水下午五点至凌晨12点供应，海珠校区要插热水卡，花都则不用钱但每日限量（可以多注册个微信小号多洗一会）
4、有早自习、晚自习吗
无早自习与晚自习。
5、有晨跑吗
无晨跑，可能隔几个星期的周一会有一次6点55要求到操场的升旗仪式。要做两遍广播体操
6、每学期跑步打卡的要求是多少公里，可以骑车吗
大一、大二下选择“体适能”选修体育课的话，可能某个老师需要你用软件跑100km
7、寒暑假放多久，每年小学期有多长
暑假大概是56天、寒假大概30-40天的样子，无小学期
8、学校允许点外卖吗，取外卖的地方离宿舍楼多远
可以点外卖，花都校区配送到楼，海珠校区过于小，从宿舍下去到窗口拿外卖也就3分钟，但是肺炎期间窗口可能过于拥挤，而且外卖去迟一秒就可能被偷。
9、学校附近有地铁站吗
海珠校区南门离广州地铁3-8号线的换乘站，客村非常近。但由于肺炎没开放。正门大概7-8分钟的步行距离。花都校区则需要坐公交车，至少25min了，可以去广州北站或者花都广场站。
10、宿舍楼有洗衣机吗
可以自行安装。
11、校园网怎么样
花都校区的校园网经常施工，两个校区的WiFi可能不稳定。
12、每天断电断网吗，几点开始断
周一-周四断网，凌晨1：37断网。
13、食堂价格贵吗，会吃出异物吗
价格不低，也不是特别高，海珠校区的食堂太小，菜式单一味道不是很好，可以点外卖。异物较常有。
14、洗澡热水供应时间
下午五点-凌晨12点。
15、校园内可以骑电瓶车吗，电池在哪能充电
不行，但海珠校区没必要，花都校区正门有租用电动车的商家。
16、宿舍限电情况
两个烧水壶用不跳闸
17、通宵自习有去处吗
图书馆会关门。无去处。
18、大一能带电脑吗
可以
19、学校里面用什么卡，饭堂怎样消费
校园卡，但校园卡除了刷食堂、体测和门禁（可以刷iqoo手机的nfc）就没什么用了。建议不使用校园卡吃食堂，因为被人捡走了容易刷光余额。可以使用微信支付和支付宝。
20、学校会给学生发银行卡吗
会，中国银行
21、学校的超市怎么样
如果没有肺炎，不关校门，在花都完全可以去润泰，在海珠则可以去沃尔玛。
22、学校的收发快递政策怎么样
目前因为肺炎要挤在窗口，热门时刻较为拥挤
23、学校里面的共享单车数目与种类如何
花都无共享单车，海珠校内无，但没必要。
24、现阶段学校的门禁情况如何，宿舍晚上查寝吗，封寝吗，晚归能回去吗
无门禁，不查寝室，几点都可以回去。

**Comment by chaosurou** [comment]
花都区和海珠区分别对应什么专业吗？有什么区别吗？

**Comment by Gdei-hero** [comment]
相关专业情况可于学校官网查询。招生简章地址：
https://web.gdei.edu.cn/zsb/2022/0624/c61a76708/pagem.htm

chaosurou ***@***.***> 于 2022年7月7日周四 19:37写道：

**Comment by championshipppp** [comment]
海珠目前附近无大超市，沃尔玛已关门
